iib/workers/config.py

```diff
@@ -46,6 +46,8 @@ class Config(object):
         'iib.workers.tasks.build_create_empty_index',
         'iib.workers.tasks.general',
     ]
+    # Path to hidden location of SQLite database
+    hidden_index_db_path = '/var/lib/iib/_hidden/do.not.edit.db'
```

Comment:
@gallettilance updated based on @yashvardhannanavati suggestion which I liked more. 
@gallettilance do you agree with this path or do you have different idea where to store the "hidden" index.db